src/autoqasm/operators/typecast.py

```diff
@@ -0,0 +1,35 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+"""Operators for int cast statements."""
+
+from typing import Any
+
+from autoqasm import types as aq_types
+
+
+def int_typecast(argument_: Any, *args, **kwargs) -> aq_types.IntVar | int:
```

Comment:
note that the Python 3.9 tests are failing due to the use of `|` on this line. To fix this, just add:
```
from __future__ import annotations
```
to the top of this file.